src/System.IO.Ports/tests/Legacy/SerialPort/ReadTimeout.cs

```diff
@@ -36,146 +36,167 @@ private enum ThrowAt { Set, Open };
 
     #region Test Cases
 
+    [ActiveIssue(15961)]
     [ConditionalFact(nameof(HasLoopbackOrNullModem))]
     public void ReadTimeout_Default_Read_byte_int_int()
```

Comment:
ok, good idea.  There's some stuff in BaudRate which measures the baud rate by timing the sending of blocks data, which is fairly sensitive.